Google.PowerShell.IntegrationTests/Storage/Storage.GcsBucket.Tests.ps1

```diff
@@ -6,6 +6,13 @@ $project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig
 # TODO(chrsmith): When Posh updates, newer versions of Pester have Should BeOfType.
 # TODO(chrsmith): Add a random suffix to bucket names to avoid collisions between devs.
 Describe "Get-GcsBucket" {
+    $script:additionalTestingProject = "quoct-test-project"
+    $script:serviceAccountInOtherTestProject = "appveyorci-testing@gcloud-powershell-testing.iam.gserviceaccount.com"
+    $script:bucketInOtherTestProject = "gcloud-powershell-additional-bucket"
+    $script:additionalKey = Resolve-Path "$PSScriptRoot\..\AdditionalServiceAccountCredentials.json" -ErrorAction Ignore
+    # Skip the service account test if the key file does not exist.
+    # Key file only exists in AppVeyor where it can be decrypted.
+    $script:skipServiceAccountTest = $null -eq $additionalKey -or (-not (Test-Path $additionalKey))
```

Comment:
Do we have another way to check the env?  I'm just worried the test could fall through the cracks.  

Can we check if we are in AppVeyor?